backend/app.js

```diff
@@ -1,7 +1,123 @@
 const express = require("express");
 const app = express();
 const port = 3000;
```

Comment:
I know that yall didnt write this but could you change the port to 3001 so that we don't run into errors with the frontend running as well